autoload/smartinput.vim

```diff
@@ -71,32 +71,39 @@ function! smartinput#define_default_rules()  "{{{2
     call add(self.names, a:name)
     let self.table[a:name] = a:urules
   endfunction
+  if exists('g:smartinput_break_undo') || v:version < 800
```

Comment:
I prefer testing a value instead of existence of a variable to change behavior like this.

Reply:
I dit it like `g:smartinput_no_default_key_mappings`, It's only check existence.

Changed to testing value at aaa808f.

Reply:
Oops, fix default to **no break undo** at 795fcf3.

Reply:
> I dit it like `g:smartinput_no_default_key_mappings`, It's only check existence.

Ah, really? So it's okay to only check existence.  It would make test code straightforward.

Reply:
Already changed to testing a value.